tests/hpc/ganglia_client.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Ganglia Test - server
```

Comment:
Can you add few words about what test actually is doing . BTW you need to change this line in any case because you copy/paste it from server :)

Reply:
sure, thanks